tutorials/algorithms/04_vqd.ipynb

```diff
@@ -163,15 +163,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 28,
    "metadata": {},
    "outputs": [],
    "source": [
     "from qiskit.algorithms.eigensolvers import VQD\n",
+    "from qiskit.algorithms.eigensolvers import NumPyEigensolver\n",
```

Comment:
This new import is not needed here right. Its imported below by the cell that uses the NumPyEigensolver